03-cleaning.Rmd

```diff
@@ -1,9 +1,9 @@
 # Data transformation
 
-(Work in progress)
+In order to start our study on NYC park events and crimes, we need to do some cleaning and transforming to the data. The first thing to do is combining all the park event information into one dataframe, since they are originally separated in for csv files. Fortunately, all event datasets contain an event id with wich we can merge all for data files into one event dataframe. As for the park crime dataset, we simply concatenated all the data file for each quarter into one crime dataframe. 
 
 ## Read Raw Data
-
+First, we read all the data files. After that, we merge all event dataframes by event id. The crime data has previously been contatenated and written out as one csv file. So we only need to read it as a dataframe. 
```

Comment:
"contatenated" => "concatenated" 

This paragraph looks weird a bit actually without the code (https://leodyl.github.io/NYCEvents/data-transformation.html#read-raw-data), I'm not sure if we need to include this "Read Raw Data" section at all.